app/src/main/res/values/strings.xml

```diff
@@ -301,13 +301,13 @@
     <string name="advanced_unlock_explanation_summary">Use advanced unlocking to open a database more easily</string>
     <string name="biometric_unlock_enable_title">Biometric unlocking</string>
     <string name="biometric_unlock_enable_summary">Lets you scan your biometric to open the database</string>
-    <string name="biometric_auto_open_prompt_title">Auto-open biometric prompt</string>
-    <string name="biometric_auto_open_prompt_summary">Automatically ask for biometric if the database is set up to use it</string>
+    <string name="biometric_auto_open_prompt_title">Auto-open biometric scan</string>
+    <string name="biometric_auto_open_prompt_summary">Biometric unlocking of all databases set up to use it</string>
     <string name="biometric_delete_all_key_title">Delete encryption keys</string>
     <string name="biometric_delete_all_key_summary">Delete all encryption keys related to biometric recognition</string>
     <string name="biometric_delete_all_key_warning">Delete all encryption keys related to biometric recognition?</string>
     <string name="unavailable_feature_text">Could not start this feature.</string>
-    <string name="unavailable_feature_version">Your Android version %1$s does not meet the minimum version %2$s required.</string>
+    <string name="unavailable_feature_version">You have Android %1$s, and need at least version %2$s.</string>
```

Comment:
Maybe replace your and you by "The Android version is..."

Reply:
That foregoes the subjective. What is the improvement?

Reply:
Simply because a device can be loaned and used by a guest user.

Reply:
And then the "you" references the owner. Or, if you will, the user now has Android version *.?